package/com.unity.formats.usd/Runtime/Scripts/IO/Geometry/MeshImporter.cs

```diff
@@ -225,7 +225,7 @@ public class GeometrySubsets {
                              SceneImportOptions options) {
       var smr = ImporterBase.GetOrAddComponent<SkinnedMeshRenderer>(go);
       if (smr.sharedMesh == null) {
-        smr.sharedMesh = new Mesh();
+        smr.sharedMesh = new Mesh {name = Guid.NewGuid().ToString()};
```

Comment:
I wonder if there's a more meaningful name we could give here (just thinking out loud, this is not a request to change it)